src/libraries/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs

```diff
@@ -1291,51 +1288,51 @@ public enum ToolboxItemFilterType
         public bool CanConvertTo(System.Type destinationType) { throw null; }
-        public virtual object ConvertFrom(System.ComponentModel.ITypeDescriptorContext context, System.Globalization.CultureInfo culture, object value) { throw null; }
-        public object ConvertFrom(object value) { throw null; }
-        public object ConvertFromInvariantString(System.ComponentModel.ITypeDescriptorContext context, string text) { throw null; }
-        public object ConvertFromInvariantString(string text) { throw null; }
-        public object ConvertFromString(System.ComponentModel.ITypeDescriptorContext context, System.Globalization.CultureInfo culture, string text) { throw null; }
-        public object ConvertFromString(System.ComponentModel.ITypeDescriptorContext context, string text) { throw null; }
-        public object ConvertFromString(string text) { throw null; }
-        public virtual object ConvertTo(System.ComponentModel.ITypeDescriptorContext context, System.Globalization.CultureInfo culture, object value, System.Type destinationType) { throw null; }
-        public object ConvertTo(object value, System.Type destinationType) { throw null; }
-        public string ConvertToInvariantString(System.ComponentModel.ITypeDescriptorContext context, object value) { throw null; }
-        public string ConvertToInvariantString(object value) { throw null; }
-        public string ConvertToString(System.ComponentModel.ITypeDescriptorContext context, System.Globalization.CultureInfo culture, object value) { throw null; }
-        public string ConvertToString(System.ComponentModel.ITypeDescriptorContext context, object value) { throw null; }
-        public string ConvertToString(object value) { throw null; }
-        public object CreateInstance(System.Collections.IDictionary propertyValues) { throw null; }
-        public virtual object CreateInstance(System.ComponentModel.ITypeDescriptorContext context, System.Collections.IDictionary propertyValues) { throw null; }
-        protected System.Exception GetConvertFromException(object value) { throw null; }
-        protected System.Exception GetConvertToException(object value, System.Type destinationType) { throw null; }
+        public virtual object? ConvertFrom(System.ComponentModel.ITypeDescriptorContext? context, System.Globalization.CultureInfo? culture, object? value) { throw null; }
+        public object? ConvertFrom(object? value) { throw null; }
+        public object? ConvertFromInvariantString(System.ComponentModel.ITypeDescriptorContext? context, string text) { throw null; }
+        public object? ConvertFromInvariantString(string text) { throw null; }
+        public object? ConvertFromString(System.ComponentModel.ITypeDescriptorContext? context, System.Globalization.CultureInfo? culture, string? text) { throw null; }
+        public object? ConvertFromString(System.ComponentModel.ITypeDescriptorContext? context, string text) { throw null; }
```

Comment:
why is text not nullable here and is in others? I don't think it accepts null, though.